### ISSUE TITLE
img alt

### ISSUE THREAD

**Issue by ghost** [KevinVanegas/prj-rev-bwfs-dasmoto#4]
https://github.com/KevinVanegas/dasmoto_project/blob/d9dce62ce009b9304071a886704329cb1d6d9be6/index.html#L15

Nice job giving all of your `img` tags an `alt` attribute with a value that adequately describes the image, just in case your website is not able to load it. 👍 

Doing so will help improve the accessibility of your website, which might be accessed by visually impaired users on screen readers-- as well as any other time that your website is simply unable to render images.
